### PR TITLE
Admin graph update 

### PR DESCRIPTION
**Pull request by colinbruce**
Admins see R1 graphs for each office
Updated controller logic to build hash with office name and 
dwp_check count by type data for each, non-digital, office.
[finishes #94093060]

### CODE REVIEW

File: app/models/office.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class Office < ActiveRecord::Base
   has_many :users
 
   scope :sorted, -> {  all.order(:name) }
+  scope :non_digital, -> { where('name != ?', 'Digital') }
```
Comment (dotemacs) [inline app/models/office.rb]: PEDANTIC HAT ON.

A style suggestion:

```ruby
where.not(name: 'Digital')
```

Reference: https://robots.thoughtbot.com/activerecords-wherenot

PEDANTIC HAT OFF.

**Comment by dotemacs** [comment]
Hello @colinbruce 

it looks good.

I just made an observation about a different scope style, see what you think of it. But it's not a merge blocker.

Thanks :smile:

**Comment by colinbruce** [comment]
:+1: for the review and :+1: for the tip.  I much prefer that syntax!

**Comment by colinbruce** [comment]
[delivers #94093060] on demo

**Pull request merged**